app/scripts/lib/errors.js

```diff
@@ -253,8 +253,10 @@ define(function (require, exports, module) {
     normalizeXHRError: function (xhr) {
       var err;
 
-      if (! xhr || xhr.status === 0) {
+      if (! xhr || xhr.status === 0 || xhr.status === 503 ) {
```

Comment:
@shane-tomlinson Could you please explain me what do you mean by normalization here?
All the other changes have been made.